### ISSUE TITLE
CondorJobService Compilation error: value requirements has incompatible type

### ISSUE THREAD

**Issue by schuhschuh** [openmole/openmole#55]
I have pulled the latest changes from the openmole master branch (1e7468ff). As the compilation failed, I deleted my ```~/.ivy2``` directory and run ```sbt clean clean-files``` in all three relevant directories of the OpenMOLE source tree (i.e., top-level, libraries/, and openmole/). Still I get the following compilation error:
```
/vol/medic01/users/as12312/Code/OpenMOLE/openmole/plugins/org.openmole.plugin.environment.condor/src/main/scala/org/openmole/plugin/environment/condor/CondorJobService.scala:64: overriding method requirements in trait CondorJobDescription of type => Option[fr.iscpif.gridscale.condor.CondorRequirement];
[error]  value requirements has incompatible type
[error]       override val requirements = environment.requirements
[error]                    ^
[info] 	 - openmole-daemon/plugins/org-openmole-core-logging_2.11-5.0-SNAPSHOT.jar
[info] 	 - openmole-daemon/plugins/com-typesafe-slick_2.11.jar
[error] one error found
```

**Comment by jopasserat** [comment]
I've got it too :)
Will fix that.

**Comment by jopasserat** [comment]
Fixed in 7ae05f72e5304c8c3fbc183380b33f8455fbc59f
